source/wp-content/themes/wporg-developer-2023/src/article-meta-github/block.php

```diff
@@ -0,0 +1,86 @@
+<?php
+namespace WordPressdotorg\Theme\Developer_2023\Article_Meta_Github;
+
+add_action( 'init', __NAMESPACE__ . '\init' );
+
+/**
+ * Registers the block using the metadata loaded from the `block.json` file.
+ * Behind the scenes, it registers also all assets so they can be enqueued
+ * through the block editor in the corresponding context.
+ *
+ * @see https://developer.wordpress.org/reference/functions/register_block_type/
+ */
+function init() {
+	register_block_type(
+		dirname( dirname( __DIR__ ) ) . '/build/article-meta-github',
+		array(
+			'render_callback' => __NAMESPACE__ . '\render',
+		)
+	);
+}
+
+/**
+ * Render the block content.
+ *
+ * @param array    $attributes Block attributes.
+ * @param string   $content    Block default content.
+ * @param WP_Block $block      Block instance.
+ *
+ * @return string Returns the block markup.
+ */
+function render( $attributes, $content, $block ) {
+	$github_handbooks = array(
+		'wpcs-handbook',
+		'blocks-handbook',
+		'rest-api-handbook',
+		'cli-handbook',
+		'adv-admin-handbook',
+	);
+
+	$post_type = $block->context['postType'];
+	$post_id = $block->context['postId'];
+
+	if (
+		! isset( $post_id )
+		|| ! isset( $post_type )
+		|| ! in_array( $post_type, $github_handbooks, true )
+	) {
+		return '';
+	}
+
+	$title = get_the_title( $post_id );
+
+	$content = do_blocks( '<!-- wp:group {"style":{"spacing":{"blockGap":"0"}},"layout":{"type":"constrained"}} --><div class="wp-block-group">' );
```

Comment:
Does this group need to be in this block?

Reply:
Ah, nevermind. I see, this is the container for the individual block.


Could we connect the first and last concatenation and `do_blocks` only once?
```
$inner_content = "<!-- wp:paragraph....";
$content .= do_blocks(
    sprintf( '<!-- wp:group {"style":{"spacing":{"blockGap":"0"}},"layout":{"type":"constrained"}} --><div class="wp-block-group">%s</div><!-- /wp:group -->'  , $inner_content ) 
)
```

Reply:
Thanks, cleaned up the markup in https://github.com/WordPress/wporg-developer/pull/388/commits/6e6761a363e11cc1f3056c479146dc9dd4c8a304